### PR TITLE
added gin sour recipe and image

### PR DESCRIPTION
**Pull request by cemuka**
I've added gin sour recipe. Also, this is my first ever pull request!

### CODE REVIEW

File: src/recipes/gin-sour.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "github": "cemuka",
+    "ingredients": [
+        "50 ml Gin", 
+        "25 ml Lemon Juice", 
+        "12 1/2 ml Simple Syrup", 
+        "Splash Egg-White"
+    ],
+    "directions": [
+        "Begin by chilling glass with ice and soda water. Add the gin, simple syrup, lemon juice and egg white to a mixing tin and dry shake for 10 seconds.",
+        " Add ice to the mixing tin and shake for a further 10 seconds.",
```
Comment (alfg) [inline src/recipes/gin-sour.json]: Seems to have an extra space on first character.

File: src/recipes/gin-sour.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        "Fill glass with ice and strain drink into glass.",
+        "Squeeze some lemon zest over drink and add lemon twist into glass."
+    ],
+    "image": "<drink-name.jpg>"
```
Comment (alfg) [inline src/recipes/gin-sour.json]: Update image property to:
```suggestion
    "image": "gin-sour.jpg"
```

**Comment by alfg** [comment]
Thanks!

**Comment by cemuka** [comment]
Wow! Now, I know PR, I guess :)

**Pull request merged**